### ISSUE TITLE
Error on Drawing activation

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#4421]
when activating the `FRAMING - 2ND FLOOR` drawing from [this](https://hub.openingdesign.com/OpeningDesign/Highland_Haven/src/commit/1be598fcd717a826d0a5fd0a2f8c74aec1c3fd33/Open/Models_and_CAD/BlenderBIM/Highland_Haven.ifc) file, i get the following error...

```
Python: Traceback (most recent call last):
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\drawing\operator.py", line 1471, in invoke
    return self.execute(context)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\drawing\operator.py", line 1480, in execute
    core.activate_drawing_view(tool.Ifc, tool.Drawing, drawing=drawing)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\core\drawing.py", line 429, in activate_drawing_view
    drawing_tool.activate_drawing(camera)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\tool\drawing.py", line 1694, in activate_drawing
    current_representation = tool.Geometry.get_active_representation(obj)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\tool\geometry.py", line 306, in get_active_representation
    if obj.data and hasattr(obj.data, "BIMMeshProperties") and obj.data.BIMMeshProperties.ifc_definition_id:
AttributeError: 'NoneType' object has no attribute 'data'
```

```

**Comment by Andrej730** [comment]
I think I've found it - the issue was that `FRAMING - 2ND FLOOR` drawing had `+ IfcAnnotation` in pset's Include which added IfcAnnotations from other drawings leading to this issue as those annotations might be not yet loaded to Blender (the drawing they're part of was not yet activated).

I've added a filter that will always filter out IfcAnnotations from other drawings in case they accidentally get there. 

But why adding IfcAnnotation to Include in the first place? 
It seems they always get included either way if they are not excluded in pset.Exclude: 
https://github.com/IfcOpenShell/IfcOpenShell/blob/e87ca960e894a1bd21682df070cd535d2e411840/src/blenderbim/blenderbim/tool/drawing.py#L1548-L1550
